k8s/kube-prometheus/conbench-flavor.jsonnet

```diff
@@ -28,6 +28,25 @@ local kp =
         },
       },
     },
+    prometheus+: {
+      prometheus+: {
+        spec+: {
+          remoteWrite: [{
+            url: '<put_a_remote_write_endpoint_url_here>',
+            basicAuth: {
+              username: {
+                name: 'kubepromsecret',
+                key: 'username',
+              },
+              password: {
+                name: 'kubepromsecret',
+                key: 'password',
+              },
+            },
+          }],
+        },
+      },
+    },
```

Comment:
For the record; CI confirms validity of this as part of the ci/minikube check on multiple levels:
- we perform JSONNET compilation (output: k8s manifest files). JSONNET compilation fails for unexpected structure/keys, etc.
- the k8s manifest files are submittes to k8s, and test-conbench-on-mk.sh confirms that after all, Prometheus deploys correctly (it would not if its config was invalid).